content/en/apps/tutorials/application-tests.md

```diff
@@ -50,10 +50,28 @@ Writing tests for CHT apps requires a good understanding of the project workflow
     ```
     After adding these scripts, you are able to run the tests by running one of these commands from the command-line:
 
-|Command|Description|
-|---|---|
-|`npm run test`|Compiles the app settings, converts the forms, then runs the tests. <sup>[recommended]|
-|`npm run unittest`|Only runs the tests.|
+    <table>
+      <tr>
+        <th><strong>Command</strong></th>
+        <th><strong>Description</strong></th>
+      </tr>
+      <tr>
+        <td>
+          <code>npm run test</code>
+        </td>
+        <td>
+          Compiles the app settings, converts the forms, then runs the tests. <sup>[recommended]</sup>
+        </td>
+      </tr>
+      <tr>
+        <td nowrap="nowrap">
+          <code>npm run unittest</code>
+        </td>
+        <td>
+          Only runs the tests.
+        </td>
+      </tr>
+    </table>
```

Comment:
please use markdown tables.  the one you replace was actually perfect!

We don't use `<sup>` in the docs.  Just add literal text as to which one you prefer and why.

```suggestion
   |Command|Description|
   |---|---|
   |`npm run test`|Compiles the app settings, converts the forms, then runs the tests (preferred solution).
   |`npm run unittest`|Only runs the tests.|
```